packages/azure-kusto-data/src/cloudSettings.ts

```diff
@@ -12,6 +12,8 @@ export type CloudInfo = {
     FirstPartyAuthorityUrl: string;
 };
 
+const AXIOS_ERR_NETWORK = axios.AxiosError.ERR_NETWORK ?? "ERR_NETWORK";
```

Comment:
I think it will lead the user to the same error - try `axios?.AxiosError?.ERR_NETWORK ?? "ERR_NETWORK"`